### ISSUE TITLE
REPORT GENERATOR FAILED in 4.5.0 (worked in 4.4.0)

### ISSUE THREAD

**Issue by heimwege** [ArnaudBuchholz/ui5-test-runner#86]
Hey hey 👋🏻 

I just updated from `4.4.0` to `4.5.0` and now I'm getting the following error when executing the test runner:

```
⠋ Generating reports
nyc merge C:\path\to\project\app\samples\target\.nyc_output C:\path\to\project\app\samples\target\.nyc_output\merged\coverage.json
┌────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
│ REPORT GENERATOR FAILED                                                                                                                                                                                    │
├───────────┬────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┤
│ generator │ C:\path\to\project\app\samples\node_modules\ui5-test-runner\src\defaults\report.js                                                                                            │
├───────────┼────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┤
│ exit code │ 4294967295                                                                                                                                                                                     │
├───────────┴────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┤
│ [Error: ENOENT: no such file or directory, open 'C:\path\to\project\app\samples\node_modules\ui5-test-runner\src\defaults\report\default.html'] {                                         │
│   errno: -4058,                                                                                                                                                                                            │
│   code: 'ENOENT',                                                                                                                                                                                          │
│   syscall: 'open',                                                                                                                                                                                         │
│   path: 'C:\\path\\to\\project\\app\\samples\\node_modules\\ui5-test-runner\\src\\defaults\\report\\default.html'                                                                         │
│ }                                                                                                                                                                                                          │
│                                                                                                                                                                                                            │
└────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
```

also the `report.html` file is no longer being created. Also I have to agree to the error message. There's no html file located at the given path in den node_modules. There's not even a `report` folder.

Used run scripts:

```
"ui:test:server": "ui5 serve --config ./ui5-test.yaml",
"ui:test:runner": "ui5-test-runner --port 8081 --url http://localhost:8080/test/testsuite.qunit.html --report-generator $/report.js $/junit-xml-report.js --report-dir ./target/QunitReport --coverage true --fail-opa-fast --coverage-settings ./.nycrc.json",
"ui:test": "start-server-and-test ui:test:server http://localhost:8080/ ui:test:runner",
```

The missing report folder can be seen: https://www.npmjs.com/package/ui5-test-runner/v/4.5.0?activeTab=code
whereas it exists in 4.4.0: https://www.npmjs.com/package/ui5-test-runner/v/4.4.0?activeTab=code

Interesting enough that the folder exists in the actual release https://github.com/ArnaudBuchholz/ui5-test-runner/releases/tag/4.5.0

**Comment by ArnaudBuchholz** [comment]
Thanks for reporting ! I am working on it

**Comment by ArnaudBuchholz** [comment]
@heimwege Problem with `.npmignore` that I now use to decide which files should be bundled in the package.

**Comment by ArnaudBuchholz** [comment]
Fixed in 4.5.1

**Comment by heimwege** [comment]
Can confirm that everything is back to normal 👍🏻
